app.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func checkingBrackets(line string) bool {
+	stack := make([]rune, 0)
+
+	for _, char := range line {
+		switch char {
+		case '(':
+			stack = append(stack, char)
+		case ')':
+			if stack[len(stack)-1] != '(' || len(stack) == 0 {
+				return false
+			}
+			stack = stack[:len(stack)-1]
+		}
+	}
+
+	return len(stack) == 0
+}
+
+func convertationFromPN(pn []string) (int, error) {
+	stack := make([]int, 0)
+
+	for _, symbol := range pn {
+		number, err := strconv.Atoi(symbol)
+		if err == nil {
+			stack = append(stack, number)
+		} else {
+			if len(stack) < 2 {
+				return -1, fmt.Errorf("Incorrect expression")
+			}
+
+			right := stack[len(stack)-1]
+			left := stack[len(stack)-2]
+			stack = stack[:len(stack)-2]
+
+			switch symbol {
+			case "+":
+				stack = append(stack, left+right)
+			case "-":
+				stack = append(stack, left-right)
+			case "*":
+				stack = append(stack, left*right)
+			case "/":
+				stack = append(stack, left/right)
+			}
+		}
+	}
+
+	if len(stack) != 1 {
+		return 0, fmt.Errorf("Incorrect expression")
+	}
+
+	return stack[0], nil
+}
+
+func convertationToPN(line string) []string {
+	stack := make([]string, 0)
+	queue := make([]string, 0)
+
+	operands := map[string]int{
+		"+": 1,
+		"-": 1,
+		"*": 2,
+		"/": 2,
+	}
+
+	arraySymbols := strings.Split(line, "")
+
+	var lastNumber bool = false
+	for _, symbol := range arraySymbols {
```

Comment:
В golang строка это последовательность байт, т.е. []byte

При работе с осмысленным текстом лучше использовать []rune - он поддерживает utf кодировки

```suggestion
	var lastNumber bool = false
	for _, symbol := range line {
```